### ISSUE TITLE
Hard to follow the updates

### ISSUE THREAD

**Issue by pihomecloud** [shinken-monitoring/mod-webui#507]
Hi,

I thought that the relases on github were following the relaese on http://shinken.io/package/webui2 but it's not the case.
The latest release is 2.3.2 but i see 2.4.1 on the official site.
What is the method to get the latest stable release from github ?

**Comment by mohierf** [comment]
The lastest release I published on shinken.in is 2.3.2. The 2.4.1 release is the one that is currently in the develop branch of this repo. This 2.4.1 release is still in tests ...
